migrations/V127__Update_Quality_Performance_Impacts_Payment.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TYPE QUALITY_PERFORMANCE_IMPACT_TYPE AS ENUM (
```

Comment:
Would it make sense to name this type generically? It seems like there could be value in naming it so it would be accessible for other types. These values don't seem to be specific to Quality Performance Impact, even though they are used to answer the question.
```suggestion
CREATE TYPE YES_NO_OTHER_TYPE AS ENUM (
```

Reply:
Yeah totally!  I didn't know what the BE paradigm was for reusing enum/always creating something unique.  Sounds like a good change

Reply:
Thanks Patrick! We often do just make something unique on the backend, but we do have a few re-usable enums as well. This one just seemed so generic that I figured its worth discussing at least :)

Reply:
Gotcha totally makes sense, nice recommendation